extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -276,29 +283,51 @@ export class InstrumentationService {
   }
 
   /**
+   * @param {!Element} el
+   * @private
+   * @return {!Object<string, string>}
+   */
+  extractAnalyticsVariables_(el) {
+    const dataSet = el.dataset;
+    const analyticsVariables = {};
+    for (const dataKey in dataSet) {
+      if (dataKey.indexOf(VARS) === 0) {
+        let variableName = dataKey.replace(VARS, '');
+        variableName = variableName[0].toLowerCase() + variableName.slice(1);
+        analyticsVariables[variableName] = dataSet[dataKey];
+      }
+    }
+    return analyticsVariables;
+  }
+
+  /**
    * @param {!Function} listener
    * @param {string} selector
    * @private
    */
   createSelectiveListener_(listener, selector) {
     return e => {
+      let el = e.target;
+      let analyticsVariables = null;
```

Comment:
you can probably just remove this line and inline the calls to extractAnalyticsVariables.